packages/labs/ssr-dom-shim/src/InternalsShim.ts

```diff
@@ -0,0 +1,155 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
```

Comment:
```suggestion
 * Copyright 2023 Google LLC
```

Reply:
> Also with regards to this, we need to add to wireit configs. @calebdwilliams I'm happy to handle any of our monorepo consistency related changes myself and push it up if you'd like.

Yeah, that would be great. I'll work on removing the unnecessary files and reverting the package-lock